### PR TITLE
Grab e2e Che logs in GH actions artifacts and allow workspace:logs to stream or not the logs

### PR DESCRIPTION
**Pull request by flacatus**
Signed-off-by: Flavius Lacatusu <flacatus@redhat.com>

### What does this PR do?
Collect workspace logs in e2e tests.
Also add ability to workspace:logs to stream logs or not by addon --follow flag.
Reff issue: https://github.com/eclipse/che/issues/19213

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [ ] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [ ] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [ ] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [ ] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

File: .github/workflows/e2e-minikube-helm.yml [hunk 1]
```diff
@@ -27,3 +27,8 @@ jobs:
         export PLATFORM=minikube
         export INSTALLER=helm
         yarn test --coverage=false --forceExit --testRegex=test/e2e/e2e.test.ts
+    - uses: actions/upload-artifact@v2
+      if: ${{ always() }}
+      with:
+        name: minikube-updates-artifacts
```
Comment (tolusha) [inline .github/workflows/e2e-minikube-helm.yml:33]: what about `Upload artifacts` ?

File: test/e2e/e2e.test.ts [hunk 2]
```diff
@@ -253,6 +254,18 @@ describe('Workspace creation, list, start, inject, delete. Support stop and dele
     })
   })
 
+  describe('Get Eclipse Che server logs', () => {
+    it('server:stop command coverage', async () => {
```
Comment (tolusha) [inline test/e2e/e2e.test.ts:258]:
```suggestion
    it('server:logs command coverage', async () => {
```

File: .github/workflows/e2e-minikube-helm.yml (hunk 1 above)
Comment (flacatus) [reply .github/workflows/e2e-minikube-helm.yml:33]: It is a zip name, will be better to use minikube-helm-artifacts. WDYT?

File: src/commands/workspace/logs.ts [hunk 3]
```diff
@@ -50,18 +51,8 @@ export default class Logs extends Command {
...
-
-      if (!workspaceRun) {
-        this.log(`Workspace ${flags.workspace} probably hasn't been started yet.`)
-        this.log('The program will keep running and collecting logs...')
-        this.log('Terminate the program when all logs are gathered...')
-      }
-    } catch (error) {
-      this.error(error)
-    }
+    this.log(`Workspace logs will be available in '${logsDirectory}'`)
```
Comment (tolusha) [inline src/commands/workspace/logs.ts:56]:
```suggestion
    if (flags.follow) {
      this.log(`Workspace logs are available in '${logsDirectory}'`)
    } else {
      this.log(`Workspace logs are being collected in '${logsDirectory}'`)    
    }  
```

**Review by tolusha — APPROVED** [review]
Find pls one remark below

**Pull request merged**